packages/hardhat-cannon/src/builder/index.ts

```diff
@@ -311,31 +362,69 @@ export class ChainBuilder {
 
     const sortedFileList = _.sortBy(
       fileList
-        .filter((n) => /^[0-9]*-.*.json$/.test(n))
+        .filter((n) => /^[0-9]*-[0-9]*.json$/.test(n))
         .map((n) => {
-          const m = n.match(/^([0-9]*)-/);
+          const m = n.match(/^([0-9]*)-([0-9]*)/);
```

Comment:
same as previous comment:
```suggestion
          const m = n.match(/^([0-9]+)-([0-9]+)/);
```

Reply:
ab33645